TYPO3.Fluid/Classes/TYPO3/Fluid/Core/Parser/SyntaxTree/ObjectAccessorNode.php

```diff
@@ -88,10 +88,22 @@ public static function getPropertyPath($subject, $propertyPath, RenderingContext
     {
         $propertyPathSegments = explode('.', $propertyPath);
         foreach ($propertyPathSegments as $pathSegment) {
+            $isHasDirectAccess = false;
             try {
-                $subject = ObjectAccess::getProperty($subject, $pathSegment);
+                if (preg_match('/^(is|has)([A-Z].*)/', $pathSegment, $matches) > 0) {
```

Comment:
more a theoretical issue probably, but this won't match all allowed php method names starting with "is" or "has". Checking with ``substring`` like it's done in ``ObjectAccess`` would be safer probably